x-pack/plugins/cases/public/components/case_view/case_view_page.tsx

```diff
@@ -146,6 +163,72 @@ export const CaseViewPage = React.memo<CaseViewPageProps>(
       }
     }, [onComponentInitialized]);
 
+    const tabs = useMemo(
+      () => [
+        {
+          id: CASE_VIEW_PAGE_TABS.ACTIVITY,
+          name: ACTIVITY_TAB,
+          content: (
+            <CaseViewActivity
+              getCaseUserActions={getCaseUserActions}
+              initLoadingData={initLoadingData}
+              ruleDetailsNavigation={ruleDetailsNavigation}
+              caseId={caseId}
+              caseData={caseData}
+              actionsNavigation={actionsNavigation}
+              showAlertDetails={showAlertDetails}
+              updateCase={updateCase}
+              fetchCaseMetrics={fetchCaseMetrics}
+              useFetchAlertData={useFetchAlertData}
+            />
+          ),
+        },
+        ...(!isMainApplication
```

Comment:
I don't think we should do this type of check. I think is better to introduce a new alert attribute in the cases context. We already have `alerts: { sync?: boolean; enabled?: boolean };`. We can change it to `alerts: { sync?: boolean; enabled?: boolean; table?: boolean };` and defaulted to `true`. Also, if `enabled` is false then we hide the alert table entirely. This way solutions can configure how they want cases. The main application already pass `{ alerts: { enabled: false } }` to disable alerts across Cases.

Reply:
I missed the fact that we had this information inside features. I'll switch to use that. I am not convinced of having a table enabled feature because it will requires us to maintain two different interfaces in the cases page view page.